index.html

```diff
@@ -0,0 +1,176 @@
+                <li><a href="about.html">About</a></li>
+                <li><a href="#product" >Product</a></li>
+                <li><a href="#partner">Partner</a></li>
+                <li><a href="about.html#past-event" class="nav-menu-cc">Events</a></li>
+            </ul>
+
+        </nav>
+    </header>
+
+    <main>
+        <section class="headline-section">
+
+            <svg fill="#000000" class="menu-icon" id="menu-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50"
+                width="100px" height="100px">
+                <path
+                    d="M 0 7.5 L 0 12.5 L 50 12.5 L 50 7.5 Z M 0 22.5 L 0 27.5 L 50 27.5 L 50 22.5 Z M 0 37.5 L 0 42.5 L 50 42.5 L 50 37.5 Z" />
+            </svg>
+            <p class="headline-greeting">Hello! Product Designers</p>
+            <h1 class="headline-title">Global association for creating quality softwrae products 2022
+            </h1>
+            <div class="headline-description">
+                <p>
+                    We work in Artifiical Intellegent, E-commerece, Agirculture, Data Mining, Robotics,
+                    Digital-Payement,
+                    cryptocurrency,
+                </p>
+            </div>
+            <h2 class="headline-date">2022-02-08 (MON) - 12 (THU)</h2>
+            <p class="headline-location">@ ASSTU center of inovation and digital transformation</p>
+        </section>
+        <section class="main-program-section">
+            <div class="title-container">
+                <h2 class="title">Main Program</h2>
+                <div class="divider"></div>
+            </div>
+
+            <div class="main-program" id="main-program">
+                <div class="program">
+                    <img src="./image/devops.png" alt="devops">
+                    <h3 class="program-title">DevOps</h3>
+                    <p class="program-description">
+
+                        We will share our experiance on DevOps and Agile software development methodology
+                    </p>
+                </div>
+                <div class="program">
+                    <img src="./image/ai.jpg" alt="machine learning">
+                    <h3 class="program-title">Machine Learning</h3>
+                    <p class="program-description">
+                        We will share our experiance on DevOps and Agile software development methodology
+                    </p>
+                </div>
+                <div class="program">
+                    <img src="./image/forum.png" alt="forum">
+                    <h3 class="program-title">Forum</h3>
+                    <p class="program-description">
+                        We will share our experiance on DevOps and Agile software development methodology
+                    </p>
+                </div>
+                <div class="program">
+                    <img src="./image/workshop.png" alt="workshop">
+                    <h3 class="program-title">Workshop</h3>
```

Comment:
Let's give padding inside the box so that the image and the text doesn't come too close to the edges. And also fix the spacing of the text.

| Your Design | Design provided |
|-------------|---------------|
| <img width="350" alt="image" src="https://user-images.githubusercontent.com/34578631/181300172-0e456373-d450-4bc9-9973-9780f55516dc.png"> | <img width="547" alt="image" src="https://user-images.githubusercontent.com/34578631/181300231-c15d1f62-deca-44a1-99bc-8e51764fe1a2.png"> |